audio-worklet/basic/audio-param-modulation.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>AudioWorklet AudioParam Modulation | Chrome WebAudio Samples</title>
+  <link rel="stylesheet" href="../../resources/base.css">
+  <script src="../lib/audio-worklet-base.js"></script>
+</head>
+<body>
+  <a href="../index.html" class="link-small">&#171; AudioWorklet home</a>
+  <h1>Demo: AudioParam Modulation</h1>
```

Comment:
This should probably have a bit more information about what the demo is actually doing with modulation. Same applies to all of the demos.  With only a title, it's really hard to know what the demo is without reading the code.